LibCore/CIODevice.h

```diff
@@ -37,7 +37,13 @@ class CIODevice : public CObject {
 
     bool can_read() const;
 
-    bool seek(signed_qword);
+    enum class SeekMode {
+        SetPosition,
+        FromCurrentPosition,
+        FromEndPosition,
+    };
+
+    bool seek(signed_qword, SeekMode mode = SeekMode::SetPosition, off_t* pos = nullptr);
```

Comment:
I usually omit parameter names that add no information beyond the type name, e.g "SeekMode = SeekMode::SetPosition"
It's definitely up to personal taste though, just thought I'd mention it once and then let it go. :)

Reply:
Hmm, not quite sure what you mean here. Can you elaborate a bit?

Reply:
I'll let the WebKit coding style document explain (since that's where I picked up the habit):

"Leave meaningless variable names out of function declarations. A good rule of thumb is if the parameter type name contains the parameter name (without trailing numbers or pluralization), then the parameter name isn’t needed. Usually, there should be a parameter name for bools, strings, and numerical types."

https://webkit.org/code-style-guidelines/#names-variable-name-in-function-decl